.github/workflows/e2e-feature-tests.yml

```diff
@@ -36,50 +53,45 @@ jobs:
         kubectl cluster-info
         kubectl get nodes
 
+    - name: Setup COSI, S3 and IAM environments
+      run: |
+        set -e -o pipefail
+        (
+          echo "=== Setup COSI Controller, CRDs and Driver ==="
+          .github/scripts/setup_cosi_resources.sh
+        ) &
+        (
+          echo "=== Loading cached S3 and IAM Docker images ==="
+          if [ -d /tmp/.docker_cache ] && [ "$(ls -A /tmp/.docker_cache 2>/dev/null)" ]; then
+            for image in /tmp/.docker_cache/*.tar; do
+              docker load -i "$image" || true  # continue on failure
```

Comment:
isn't there an action that can load from the cache or save images to cache  ?

Reply:
Yes, there is, but I can't use them in a script when doing things in parallel.
Ideally, we would want to use the action, but GitHub doesn't allow parallel actions on the same machine yet.
What do you think?

Reply:
Oh that's unlucky

Reply:
yes, it is. But saving 1 minute with every CI run, it added up, so changed this.